t/v3-security.t

```diff
@@ -0,0 +1,516 @@
+use Mojo::Base -strict;
+use Test::Mojo;
+use Test::More;
+
+use Mojolicious::Lite;
```

Comment:
Maybe move this line below "skip_all", since it's used together with defining the lite app routes.

Reply:
skip_all placed here by analogy with code in [v3-nullable.t](https://github.com/jhthorsen/mojolicious-plugin-openapi/blob/master/t/v3-nullable.t#L1-L7) and v3.t. Do you mean, there is a need to alter all three files?
Or there is some specific in v3-security.t, which I don't see?

Reply:
No, I'm just not perfect 🤣hehe

Only need to change it one place, and I can fix the others. (You don't have to clean up my mess)

Reply:
I'm not sure that I understand you correctly. I wrap skip_all into BEGIN block (cause **use** runs in BEGIN) and place it over **use Mojolicious::Lite;**.

Reply:
Hmm, looks like it going to failing test. Rollback it tomorrow.

Reply:
By the way, Why not to install Yaml::XS into travis?

Reply:
Failed tests are not from my PR.